roles/kubernetes/client/tasks/main.yml

```diff
@@ -79,7 +79,7 @@
   copy:
     content: |
       #!/bin/bash
-      kubectl --kubeconfig=${BASH_SOURCE%/*}/admin.conf $@
+      ./kubectl --kubeconfig=${BASH_SOURCE%/*}/admin.conf $@
```

Comment:
should be an absolute path

Reply:
The kubectl is relative to kubectl.sh. Don't know if it makes sense to use an absolute path here.

Reply:
Correct, we don't know where the user cloned the repository so we do not know the correct absolute path.

Reply:
@ant31 can you approve this?

Reply:
kubectl is fetched in the previous task to `{{ artifacts_dir }}/kubectl`, can't it be used here?